frontend/src/features/manageProfile/components/forms/companyInfo/CompanyInfoForm.tsx

```diff
@@ -157,7 +157,7 @@ export const CompanyInfoForm = memo(
                     invalidDBALength(1, 150),
                 },
               },
-              label: "DBA",
+              label: "Doing Business As",
```

Comment:
Does it need to be changed here too?
https://github.com/bcgov/onroutebc/blob/main/frontend/src/features/wizard/subcomponents/CompanyInformationWizardForm.tsx#L66

Reply:
@krishnan-aot I have raised this with Michael and he is happy with the change just being at Company Profile, @zgong-gov did raise the same point however so I will confirm with him tomorrow that he is okay to merge with the change in just the one place

Reply:
If Michael is happy with just one place, then we can merge this.